ReactWindows/ReactNative.Shared/Bridge/ReactContext.cs

```diff
@@ -0,0 +1,349 @@
+﻿using ReactNative.Bridge.Queue;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// Abstract context wrapper for the React instance to manage
+    /// lifecycle events.
+    /// </summary>
+    public class ReactContext : IReactContext
```

Comment:
Nitpick, but can you squash your commits so this shows up as a move instead?

---
In reply to: [80768117](https://github.com/ReactWindows/react-native-windows/pull/713#discussion_r80768117) [](ancestors = 80768117)